content/well-architected-framework/docs/docs/secure-systems/infrastructure/zero-trust-security.mdx

```diff
@@ -0,0 +1,129 @@
+trust security model and how it can protect your organization.
+
+<VideoEmbed url="https://www.youtube.com/watch?v=FCWl-1Q-GIQ"/>
+
+Throughout this series, you will see how HashiCorp tools work together, so you
+can create a strong security model following zero trust principles across your
+infrastructure.
+
+1. **Terraform manages infrastructure changes:**
+
+   - Engineers modify infrastructure through code review.
+   - [CI/CD](/well-architected-framework/define-and-automate-processes/automate/cicd)
+     systems use Vault-generated cloud credentials.
+   - No human has direct cloud console access.
+
+1. **Vault manages secrets and identities:**
+
+   - Manages static and dynamic secrets.
+   - Creates a trusted identity platform to authenticate and authorize users and
+     devices.
+   - Issues just-in-time credentials for infrastructure access.
+   - Manages certificates for both PKI and certificate-based authentication.
+
+1. **Boundary and Vault provide access to infrastructure:**
+
+   - Support teams access infrastructure through Boundary.
+   - Vault provides just-in-time credentials for each session.
+   - All access is temporary, audited, and automatically expires.
+
+1. **Consul enables network security:**
+
+   - API gateways and ingress controllers manage north-south traffic with identity-based policies.
+   - Service mesh provides automatic mTLS encryption and service-to-service authorization.
+   - Service intentions define which services can communicate, providing
+     micro-segmentation that blocks unauthorized lateral movement.
+
+1. **Waypoint allows developer self-service:**
+
+   - Developers deploy through standardized workflows without infrastructure access.
+   - Platform teams define secure deployment patterns that developers automatically follow.
+   - Reduces the number of people requiring direct infrastructure permissions.
+
+Combining these tools creates a strong foundation for a zero-trust model where
+access to infrastructure is both secure and auditable without sacrificing
+operational efficiency.
+
+<VideoEmbed url="https://www.youtube.com/watch?v=lT11tGaEC3s"/>
+
+Other services to complete your zero trust architecture include:
+
+- **Identity providers** - Centralized [identity and access management
+  systems](/well-architected-framework/secure-systems/identity-access-management/define-access-requirements)
+  (Okta, Azure AD, Ping Identity) that integrate with Vault to provide SSO and
+  MFA capabilities.
+- **SIEM solutions** - Security Information and Event Management (SIEM)
+  solutions (Splunk, ELK Stack, Grafana) that collect and analyze logs
+  from Vault, Boundary, Consul, and other systems to provide real-time
+  monitoring and alerting.
+- **Endpoint security solutions** - Tools that ensure devices accessing the
+  network meets security standards, such as antivirus software, firewalls, and
+  endpoint detection and response (EDR) solutions.
+
+HashiCorp resources:
```

Comment:
Should we add a "Get started with Terraform" here as well since it's covered in a few places?